packages/core-commands/src/site-editor-global-styles-commands.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * WordPress dependencies
+ */
+import { useCommandLoader } from '@wordpress/commands';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { getPath, addQueryArgs } from '@wordpress/url';
+import { styles, external } from '@wordpress/icons';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from './lock-unlock';
+
+const { useHistory } = unlock( routerPrivateApis );
+
+const getGlobalStylesNavigationCommands = () =>
+	function useGlobalStylesNavigationCommands() {
+		const history = useHistory();
+		const isSiteEditor = getPath( window.location.href )?.includes(
+			'site-editor.php'
+		);
+
+		const { isBlockBasedTheme, canCreateTemplate } = useSelect(
+			( select ) => {
+				const { getCurrentTheme, canUser } = select( coreStore );
+
+				return {
+					isBlockBasedTheme: getCurrentTheme()?.is_block_theme,
+					canCreateTemplate: canUser( 'create', {
+						kind: 'postType',
+						name: 'wp_template',
+					} ),
+				};
+			},
+			[]
+		);
+
+		const commands = useMemo( () => {
+			// Only show site editor commands to users who can access it and in block themes
+			if ( ! canCreateTemplate || ! isBlockBasedTheme ) {
+				return [];
+			}
+
+			const result = [];
+
+			// Go to Styles command
+			result.push( {
+				name: 'core/go-to-styles',
+				label: __( 'Go to Styles' ),
```

Comment:
```suggestion
				label: __( 'Go to: Styles' ),
```

Following #71476, it would be a good idea to change the command name from 'Go to {something}' to 'Go to: {something}'.